doc/categories.json

```diff
@@ -0,0 +1,580 @@
+{
```

Comment:
It might be better to have a more informative name for this file, such as `categories_map.json`.